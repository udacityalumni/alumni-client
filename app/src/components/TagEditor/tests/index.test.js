import TagEditor from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<TagEditor />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <TagEditor
        tags={[]}
        selectedTags={[]}
        onChangeValue={e => e}
      />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
