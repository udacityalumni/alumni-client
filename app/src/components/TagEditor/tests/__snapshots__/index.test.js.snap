exports[`<TagEditor /> should render with default props 1`] = `
<Box
  announce={false}
  direction="column"
  focusable={true}
  pad="none"
  responsive={true}
  tag="div">
  <CreatableSelect
    filterOptions={[Function filterOptions]}
    isOptionUnique={[Function isOptionUnique]}
    isValidNewOption={[Function isValidNewOption]}
    menuRenderer={[Function menuRenderer]}
    multi={true}
    newOptionCreator={[Function newOptionCreator]}
    onChange={[Function onChangeValue]}
    options={false}
    promptTextCreator={[Function promptTextCreator]}
    shouldKeyDownEventCreateNewOption={[Function shouldKeyDownEventCreateNewOption]}
    value={Array []} />
</Box>
`;
